enterprise/payara.micro/src/org/netbeans/modules/fish/payara/micro/project/MicroActionsProvider.java

```diff
@@ -65,78 +36,40 @@
         service = MavenActionsProvider.class,
         projectType = MAVEN_WAR_PROJECT_TYPE
 )
-public class MicroActionsProvider implements MavenActionsProvider {
+public final class MicroActionsProvider extends AbstractMavenActionsProvider {
 
     @StaticResource
     private static final String ACTION_MAPPINGS = "org/netbeans/modules/fish/payara/micro/project/resources/action-mapping.xml";
-        
-    private final AbstractMavenActionsProvider actionsProvider = new AbstractMavenActionsProvider() {
```

Comment:
Not needed as we provide unique actions in our mapping file. The MicroActionConvertor would convert standard actions to micro actions when they are supported by the project.